### PR TITLE
feat: user can delete tags.

### PR DESCRIPTION
**Pull request by CheoR**
# Description

Only staff can delete tags.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# Testing Instructions

`git fetch --all`
`git checkout cr-tags-delete`
`npm start`

See server-side pull request for backend instructions.

When logged in as staff:

After logging in, click on `Tag Management` link and delete a tag.
Tag list should update.

When logged in as non-staff:

After logging in, click on `Tag Management` link there should be no delete option.

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings or errors
- [X] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by phillip-phan-imprivata — APPROVED** [review]
i can indeed delete tags

File: src/components/tags/TagList.js [hunk 1]
```diff
@@ -4,39 +4,58 @@ import { TagContext } from "./TagProvider"
...
+        <ul className="tags_list">
+          {
+            tags.map((tag) => {
 
-     {
-       loggedInUser
-       ?
+              return <li key={tag.id} className="tags_list--item">
+                <div className="tag_label">{ tag.label }</div>
+                { tag.isStaff && <div>
```
Comment (notdanross) [inline src/components/tags/TagList.js]: When my client requests tags from the server, the objects don't have a property of `isStaff`. Is this an error for me only, or is something missing from the code?

**Review by notdanross — CHANGES_REQUESTED** [review]

File: src/components/tags/TagList.js (hunk 1 above)
Comment (CheoR) [reply src/components/tags/TagList.js]: @Daniel-L-Ross  It was was the latest main branch. isStaff is set on Login.js

**Review by notdanross — APPROVED** [review]
Good work. Only able to delete as admin.

**Pull request merged**